analysis/selections/simpTight_8hits.json

```diff
@@ -0,0 +1,28 @@
+{
+    "eSum_lt"  : {
+	"cut"  : 1.9,
```

Comment:
simpTightL1L1 and this have different cut values. Just want to make sure you really intended for that.